### PR TITLE
feat(plan-manager): added downgrade pop-up

### PR DESCRIPTION
**Pull request by evgenibir**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

without issue

### ✅ Checklist

- Created downgrade pop-up

### 🙈 Screenshots

image

### CODE REVIEW

File: src/pages/dho/Plan.vue [hunk 1]
```diff
@@ -190,6 +201,36 @@ export default {
 
 <template lang="pug">
 .page-plan(v-if="!loading")
+  q-dialog(:value="downgradePopUp" persistent)
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/pages/dho/Plan.vue]: Ready. @pugakn

File: src/pages/dho/Plan.vue [hunk 2]
```diff
@@ -190,6 +202,7 @@ export default {
 
 <template lang="pug">
 .page-plan(v-if="!loading")
+  DowngradePopUp(:value="downgradePopUp" @activatePlan="activatePlan" @hidePopUp="downgradePopUp = false")
```
Comment (arsenijesavic) [inline src/pages/dho/Plan.vue:205]: @evgenibir We have state var to manage diffrent states. I suggested you change downgradePopUp bool to `state: DOWNGRADING` so we follow the state. Additionally you can add computed vars if you want something like `isDowgradePopUpOpen () { return state === 'DOWNGRADING'}` but i think its not necessary

**Review by arsenijesavic — CHANGES_REQUESTED** [review]

File: src/components/plan/downgrade-pop-up.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<script>
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/components/plan/downgrade-pop-up.vue:1]: @arsenijesavic fixed!

File: src/pages/dho/Plan.vue (hunk 2 above)
Comment (evgenibir) [reply src/pages/dho/Plan.vue]: @arsenijesavic fixed!

Comment (arsenijesavic) [reply src/pages/dho/Plan.vue]: awsome!

File: src/pages/dho/Plan.vue [hunk 4]
```diff
@@ -190,6 +203,7 @@ export default {
 
 <template lang="pug">
 .page-plan(v-if="!loading")
+  DowngradePopUp(:value="isDowngradePopUpOpen" @activatePlan="activatePlan" @hidePopUp="state = 'BILLING'")
```
Comment (arsenijesavic) [inline src/pages/dho/Plan.vue:206]: @evgenibir just small change if you can lowercase the component name as we always used that convetion :)

**Review by arsenijesavic — CHANGES_REQUESTED** [review]

**Review by arsenijesavic — APPROVED** [review]

**Pull request merged**